src/collapse/mod.rs

```diff
@@ -1,63 +1,39 @@
-pub mod bpftrace;
-pub mod perf;
+mod bpftrace;
+mod perf;
 
+pub use bpftrace::Bpftrace;
+pub use perf::{Perf, PerfOptions};
```

Comment:
I think I prefer these to be accessed through the `mod`s (so leave them as `pub`) rather than re-exporting them here. That way we avoid the number of types swelling as we add more collapsers.

Reply:
👍